daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md

```diff
@@ -0,0 +1,156 @@
+---
+type: docs
+title: "How-To: Schedule and handle triggered jobs"
+linkTitle: "How-To: Schedule and handle triggered jobs"
+weight: 2000
+description: "Learn how to use the jobs API to schedule and handle triggered jobs"
+---
+
+Now that you've learned what the [jobs building block]({{< ref jobs-overview.md >}}) provides, let's look at an example of how to use the API. The code example below describes an application that schedules jobs for a database backup application and handles them at trigger time, also known as the time the job was sent back to the application because it reached it's dueTime.
+
+<!-- 
+Include a diagram or image, if possible. 
+-->
+
+## Start the Scheduler service
+
+When you [run `dapr init` in either self-hosted mode or on Kubernetes]({{< ref install-dapr-selfhost.md >}}), the Dapr Scheduler service is started.
+
+## Set up the Jobs API
+
+In your code, set up and schedule jobs within your application.
+
+{{< tabs "Go" >}}
+
+{{% codetab %}}
+
+<!--go-->
+
+The following Go SDK code sample schedules the job named `prod-db-backup`. Job data is housed in a backup database (`"my-prod-db"`) and is scheduled with `ScheduleJobAlpha1`, 
+providing the job `data` which is the backup `task` name and `metadata` including the location and database name. The job is scheduled with a `schedule` set and the amount of `repeats`
+desired, meaning there is a max amount of times the job should be triggered and sent back to the app. At trigger time, so `@every 1s` according to the `schedule`, this job will be 
+triggered and sent back to the application up to the max repeats set which is `10`. At the trigger time, the `prodDBBackupHandler` function is called executing the 
+desired business logic for this job at trigger time. For example:
+
+```go
+package main
+
+import (
+    //...
+
+	daprc "github.com/dapr/go-sdk/client"
+	"github.com/dapr/go-sdk/examples/dist-scheduler/api"
+	"github.com/dapr/go-sdk/service/common"
+	daprs "github.com/dapr/go-sdk/service/grpc"
+)
+
+func main() {
+    // Initialize the server
+	server, err := daprs.NewService(":50070")
+    // ...
+
+	if err = server.AddJobEventHandler("prod-db-backup", prodDBBackupHandler); err != nil {
+		log.Fatalf("failed to register job event handler: %v", err)
+	}
+
+	log.Println("starting server")
+	go func() {
+		if err = server.Start(); err != nil {
+			log.Fatalf("failed to start server: %v", err)
+		}
+	}()
+    // ...
```

Comment:
The spacing is showing up weird here, but looks fine in my IDE

Reply:
it looks fine when I build locally too!